include/thread_pool/worker.hpp

```diff
@@ -47,19 +49,19 @@ class Worker
     void stop();
 
     /**
-     * @brief post Post task to queue.
+     * @brief tryPost Post task to queue.
      * @param handler Handler to be executed in executing thread.
      * @return true on success.
      */
     template <typename Handler>
-    bool post(Handler&& handler);
+    bool tryPost(Handler&& handler);
 
     /**
-     * @brief steal Steal one task from this worker queue.
-     * @param task Place for stealed task to be stored.
+     * @brief tryGetLocalTask Get one task from this worker queue.
+     * @param task Place for the obtained task to be stored.
      * @return true on success.
      */
-    bool steal(Task& task);
+    bool tryGetLocalTask(Task& task);
```

Comment:
Not sure about the name here - trySteal?

Reply:
I didn't like the name "steal" for this method because I thought it conflated the idea of "stealing" and the idea of "getting stolen from" relative to the calling object. Its the latter that is happening whereas to me the name suggests the former.